cmd/src/setup.rs

```diff
@@ -69,18 +71,9 @@ pub fn initial_logger(config: &TiKvConfig) {
                 e
             );
         });
-        if config.slow_log_file.is_empty() {
-            logger::init_log(
```

Comment:
why change this logic?

Reply:
I think these changes should be reverted.

Reply:
I think it's okay, it combines all drainer to here? https://github.com/tikv/tikv/pull/7358/files#diff-1489fd33ea1fb2aa1c8ab49ede84ad07R119

Reply:
I see. I misread the logic.